website/pages/chapters/assessing_performance.js

```diff
@@ -255,31 +314,45 @@ export default function AssessingPerformance() {
         <p>
           Well in some sense, you want as big of a test set as possible since
           the more examples in your test set, the better estimate of the true
-          error you will get. You can think of the extreme case where your test
-          set contains all possible input/outputs, then you will exactly be able
+          error you will get. Imagine the extreme case where the test set
+          contains all possible input/outputs, then you will exactly be able
+          <MatthewComment>
+            This is not necessarily true? I think you want your test set to
+            represent the underlying joint distribution{" "}
+            <IM math={`(x,y) \\sim P(x, y).`} />
+          </MatthewComment>{" "}
```

Comment:
Yeah, I could call this out but I thought it would be better not to. There is *always* the underlying assumption (which is almost never actually true) that your sets are representative of the true distribution, otherwise the whole endeavor is pointless.

Reply:
Fair fair, this one was more for my curiosity more than anything :p